prime-router/src/main/java/com/function/Function.java

```diff
@@ -0,0 +1,43 @@
+package com.function;
+
+import com.microsoft.azure.functions.ExecutionContext;
+import com.microsoft.azure.functions.HttpMethod;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.HttpResponseMessage;
+import com.microsoft.azure.functions.HttpStatus;
+import com.microsoft.azure.functions.annotation.AuthorizationLevel;
+import com.microsoft.azure.functions.annotation.FunctionName;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
+
+import java.util.Optional;
+
+/**
+ * Azure Functions with HTTP Trigger.
+ */
+public class Function {
+    /**
+     * This function listens at endpoint "/api/HttpExample". Two ways to invoke it using "curl" command in bash:
+     * 1. curl -d "HTTP Body" {your host}/api/HttpExample
+     * 2. curl "{your host}/api/HttpExample?name=HTTP%20Query"
+     */
+    @FunctionName("HttpExample")
+    public HttpResponseMessage run(
+            @HttpTrigger(
```

Comment:
Why did you resolve this question without a response? Please talk to me.

Reply:
oh - sorry - thought I did; or at least in one section did -- this was code that was apparently added by VSCode...I removed it (or at least thought I did) on the next check in.  I think I added them all in one